src/science/roughness/cable_roughness.F90

```diff
@@ -22,17 +23,22 @@
 
 MODULE cable_roughness_module
 
-USE cable_phys_constants_mod, ONLY : CCSD   => CSD 
-USE cable_phys_constants_mod, ONLY : CCRD   => CRD 
-USE cable_phys_constants_mod, ONLY : CCCD   => CCD 
-USE cable_phys_constants_mod, ONLY : CCCW_C => CCW_C 
-USE cable_phys_constants_mod, ONLY : CUSUHM  => USUHM 
-USE cable_phys_constants_mod, ONLY : CVONK   => VONK
-USE cable_phys_constants_mod, ONLY : CA33    => A33 
-USE cable_phys_constants_mod, ONLY : CCTL    =>  CTL  
-USE cable_phys_constants_mod, ONLY : CZDLIN  => ZDLIN
-USE cable_phys_constants_mod, ONLY : CCSW    => CSW
-USE cable_phys_constants_mod, ONLY : CGRAV   => GRAV
+#ifdef UM_CBL
```

Comment:
`#ifdef` okay for now - but surely (for this one) we can get round the need for this by modifying the files names in the offline/coupled codebases to be the same.